docs/faqs.md

```diff
@@ -204,6 +205,16 @@ As a result, all calls to `tools.check_min_cppstd` must be guarded by a check fo
 We defer adding older versions without a direct requirement. We love to hear why in the opening description of the PR.
 This is for historical reasons, when older versions were permitted the overwhelming majority received zero downloads and were never used by the community while still increasing the burden on the build system.
 
+## What is the policy for removing older versions of a package?
+
+Keeping many older versions can be a problem, as over time they may become incompatible with newer versions of the package's Python code and/or dependencies. They also become downloaded less often than newer versions, and yet continue to consume CI resources during Pull Requests.
+
+Thus, removing older versions from
+`config.yml` and `conandata.yml` is totally fine. The respective binary packages will not be removed from Conan Center, but they will not receive new updates, as
+they are not listed to be built.
```

Comment:
I agree with @Croydon... I find the language ("totally fine") is a little liberal. How about this?

```suggestion
Given a technical limitations and/or incompatibilities emerging from infrastructure changes, removing older versions from `config.yml` and `conandata.yml` may be permitted. The respective binary packages will not be removed from Conan Center, but they will not receive new updates, as they are not listed to be built.
```